diff --git a/public/mathlib/mathlib.h b/public/mathlib/mathlib.h
index 4769bbab..af50820a 100644
--- a/public/mathlib/mathlib.h
+++ b/public/mathlib/mathlib.h
@@ -328,7 +328,7 @@ void inline SinCos( float radians, float *sine, float *cosine )
 		fstp DWORD PTR [eax]
 	}
 #elif defined( _LINUX ) || defined ( __APPLE__ )
-	register double __cosr, __sinr;
+	double __cosr, __sinr;
  	__asm __volatile__
     		("fsincos"
      	: "=t" (__cosr), "=u" (__sinr) : "0" (radians));
